Shared_Il2cpp/ClothingStateMenuPlugin.cs

```diff
@@ -126,14 +135,25 @@ private bool RefreshCharacterList()
             return anyChara;
         }
 
-
         private void Update()
         {
             if (Keybind.Value.IsDown())
                 ShowInterface = !ShowInterface;
 
             if (_showInterface)
             {
+#if AC
+                // The window won't auto close when the plugin is used in game and the player
+                // don't close it after use it. There is a ocassion that I don't like if leave
+                // open on and H Scene the window will show during the stats update screen.
+                // I close did it using a hook to HScene.update_ for AC but decided is to
+                // complicated just for that. Removed the code.
+                if (GameUtilities.GetCurrentHumans().ToArray().Length == 0)
```

Comment:
This was should be faster.
```suggestion
                if (GameUtilities.GetCurrentHumans().Any())
```